src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1704,7 +1704,9 @@ def pick_strategies(strategy_pool, fuzzer_path, corpus_directory,
     fuzzing_strategies.append(strategy.MUTATOR_PLUGIN_STRATEGY.name)
 
   if (strategy_pool.do_strategy(strategy.MUTATOR_PLUGIN_RADAMSA_STRATEGY) and
-      use_radamsa_mutator_plugin(extra_env)):
+      use_radamsa_mutator_plugin(extra_env)
+     ) and not fuzzing_strategies.contains(
```

Comment:
Nit: keep this close paren before the colon.

Reply:
Done